test/unit/utils.test.ts

```diff
@@ -0,0 +1,61 @@
+import { buildURL } from '../../src/utils';
+
+describe('buildURL general use case', () => {
+  it('if a URL is passed, return error', () => {
```

Comment:
Test description should be backward, and read like:

`it('returns error if a URL is passed'`